Core/App/AppInstaller.php

```diff
@@ -338,11 +337,10 @@ private function testMysql($dbData)
             if ($connection->query($sqlCrearBD)) {
                 return true;
             }
-
-            $this->miniLog->critical((string) $connection->connect_error);
         }
 
         $this->miniLog->critical($this->i18n->trans('cant-connect-database'));
+        $this->miniLog->critical((string) $connection->connect_errno . ': ' . $connection->connect_error);
```

Comment:
With that and the change of original if to oposite condition, we avoid else and also put two times the same critical message.